.github/workflows/make.yml

```diff
@@ -51,24 +51,32 @@ jobs:
 
   windows_build:
     runs-on: windows-latest
+    strategy:
+      matrix:
+        include: [
+          {installs: "MINGW32", pkg: "mingw-w64-i686-gcc" },
+          {installs: "MINGW64", pkg: "mingw-w64-x86_64-gcc" },
+        ]
 
     defaults:
       run:
         shell: msys2 {0}
 
     steps:
-    - uses: actions/checkout@v3
-    - uses: msys2/setup-msys2@v2
+    - name: Setup msys2
+      uses: msys2/setup-msys2@v2
       with:
+        msystem: ${{ matrix.installs }}
         update: true
-        install: autoconf automake libtool gcc make groff zip dos2unix
+        install: base-devel git autoconf automake libtool groff zip dos2unix ${{ matrix.pkg }}
+    - uses: actions/checkout@v3
     - name: Find Git version
       id: version
       run: |
         export VERSION=$(git rev-parse --short HEAD)
         echo "VERSION=$VERSION-win32" >> $GITHUB_OUTPUT
```

Comment:
Suffix should be different for 64-bit version?

Reply:
I don't think so

Reply:
I misunderstood that thing. fixing..